src/platform/export/fileConverter.ts

```diff
@@ -107,10 +108,17 @@ export class FileConverter implements IFileConverter {
         }
```

Comment:
@rchiodo this method was identical in the child class, except for the line 110, we were adding a new argument.
Tomorrow if we have more formats then we need to copy code from the parent class into this child class, which isn't good